chezbetty/models/user.py

```diff
@@ -376,9 +376,11 @@ def groupfinder(userid, request):
     user = User.from_uniqname(userid)
     if user.role == "user":
         return ["user",]
+    elif user.role == "volunteer":
+        return ["user","volunteer"]
     elif user.role == "manager":
-        return ["user","manager"]
+        return ["user","volunteer","manager"]
     elif user.role == "administrator":
-        return ["user","manager","admin","serviceaccount"]
+        return ["user","volunteer","manager","admin","serviceaccount"]
```

Comment:
I don't think we need to create a "volunteer" group here, particularly because it currently isn't being used anywhere. So all that is needed is:

```python
    elif user.role == "volunteer":
        return ["user"]
```

Reply:
Done.

Reply:
You also have to remove the "volunteer" from the other roles.

Reply:
Wouldn't removing "volunteer" from all other roles mean that groupfinder would never work as a search for "volunteer"?  What benefit does this give?

Reply:
So `groupfinder` does a mapping from user roles to groups the user with that role is a member of. So by adding `volunteer` to any of the returned arrays that is effectively creating a new group. This code:

https://github.com/um-cseg/chez-betty/blob/e5ede65414df8be28807f8206dacdcb488644845/chezbetty/models/model.py#L46-L54

then converts those groups into what permissions the users in that group have. Since nothing was added to the ACLs, being in group "volunteer" has no practical purpose. And at that point it doesn't really help to add it.

Reply:
Got it; I removed all references and made searching "volunteer" return the same as "user" (without the random comma).